core/src/main/java/io/questdb/std/str/ByteCharSequence.java

```diff
@@ -34,12 +34,15 @@
  * UTF8-encoded on-heap char sequence.
  */
 public class ByteCharSequence extends AbstractCharSequence implements ByteSequence {
-    private final byte[] bytes;
+    private byte[] bytes;
 
     public ByteCharSequence(byte[] bytes) {
         this.bytes = bytes;
     }
 
```

Comment:
Are these changes used anywhere ?

Reply:
there used to. but not anymore in this latest version. thanks!